### ISSUE TITLE
"Admin Dataverse" - Improve This Display Name, Other Attributes

### ISSUE THREAD

**Issue by mheppler** [IQSS/dataverse#1696]
- [x] Change “Admin Dataverse” display name to “Dataverse Admin”. (Easy to edit in /script/api/data/user-admin.json file.) [Done by Merce] 34420dd253f5fae0536672f6c3be04956adf9aa9
- [x] Remove “phone” because it is not even in the Sign Up form. [Done by Merce] 34420dd253f5fae0536672f6c3be04956adf9aa9
- [x] Change “admin@mailinator.org” to “support@dataverse.org”. [Done by Merce] 34420dd253f5fae0536672f6c3be04956adf9aa9
- [x] Review new “installer” specific email address, rather than "support"?
- [x] Add Installation Guide note to change this name, and email, before installing. [Done by Phil]
- [x] “Dataverse Admin” == “IQSS Admin”, in migration, the “id” reference can change this. Change to “Sonia Barbosa” in post-migration cleanup script?

**Comment by pdurbin** [comment]
@kcondon I believe I've addressed the remaining concerns above in my pull request at #2895 

Here's where you can preview the change: http://guides.dataverse.org/en/2884-install-guide-reorg/installation/installation-main.html#superuser-account

Related is documenting the "system" email at #2663 which is of course different than the email address of the superuser created by the installer.

Passing to QA.

**Comment by mheppler** [comment]
@kcondon, @pdurbin, @scolapasta 
Gentlemen. I have confirmed that this issue can be closed. Phil and I have reviewed the two open issues that were in the original checklist, "email in the installer" and "Dataverse Admin". Phil, confirmed that the installer email issue is resolved. I discussed the second "Dataverse Admin" issue with Gustavo and confirmed that there is no longer a problem with the old "IQSS Admin" and no longer an issue with migrated data.
